kernel/constr.mli

```diff
@@ -25,9 +25,7 @@ type metavariable = int
 type case_style = LetStyle | IfStyle | LetPatternStyle | MatchStyle
   | RegularStyle (** infer printing form from number of constructor *)
 type case_printing =
-  { ind_tags : bool list; (** tell whether letin or lambda in the arity of the inductive type *)
-    cstr_tags : bool list array; (** tell whether letin or lambda in the signature of each constructor *)
-    style     : case_style }
+  { style     : case_style }
```

Comment:
Is it useful to keep this record? ie do we expect to add some fields to it someday?

Reply:
It doesn't cost much if we want to add other flags. I'd rather keep it.